.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+**This issue tracker is a tool to address bugs and feature requests in
+Flask-SQLAlchemy itself. Please use the [#pocoo IRC channel on freenode](http://flask.pocoo.org/community/irc/)
+or [Stack Overflow](https://stackoverflow.com/questions/tagged/flask-sqlalchemy?sort=linked) for general questions about using Flask-SQLAlchemy
+or issues not related to Flask-SQLAlchemy.**
+
+If you'd like to report a issue in Flask-SQLAlchemy, fill out the template below.
+Provide any extra information that may be useful / related to your problem.
+Ideally, create a [minimal, complete, and verifiable example (MCVE)](https://stackoverflow.com/help/mcve),
+which helps us understand the problem and helps check that it is not caused
+by something in your code.
+
+Flask-SQLAlchemy is a thin wrapper that combines [Flask](https://github.com/pallets/flask) and [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy). Please
+make sure your issue is **actually with Flask-SQLAlchemy and not SQLAlchemy** before
+submitting. Consider checking the traceback to see if the error is coming from SQLAlchemy,
+or checking if your issue has already been reported as a [SQLAlchemy issue on GitHub](https://github.com/sqlalchemy/sqlalchemy/issues).
+
+Also, please be kind - this project is maintained by volunteers. We appreciate
+you taking the time to provide the information needed to address your issue.
+
+---
+
+### Expected Behavior
+
+Tell us what should happen.
+
+```python
+Paste a minimal example that causes the problem.
+```
+
+### Actual Behavior
+
+Tell us what happens instead.
+
+```pytb
+Paste the full traceback if there was an exception.
+```
+
+### Environment
+
+* Operating system:
+* Python version:
+* Flask-SQLAlchemy version:
+* SQLAlchemy version:
```

Comment:
@lbeaufort IMO, We should have a section for `How To Reproduce the error`, else lgtm